build-tools/packages/build-tools/src/repoPolicyCheck/handlers/npmPackages.ts

```diff
@@ -585,6 +585,47 @@ export const handlers: Handler[] = [
             return { resolved: resolved };
         },
     },
+    {
+        name: "npm-package-json-prettier",
+        match,
+        handler: (file) => {
+            let json;
+
+            try {
+                json = JSON.parse(readFile(file));
+            } catch (err) {
+                return "Error parsing JSON file: " + file;
+            }
+
+            const hasPrettierProperty = Object.prototype.hasOwnProperty.call(json, "prettier");
+            const hasPrettierFixProperty = Object.prototype.hasOwnProperty.call(
+                json,
+                "prettier:fix",
+            );
+            const hasFormatProperty = Object.prototype.hasOwnProperty.call(json, "format");
+
+            if (!(hasPrettierProperty && hasPrettierFixProperty && hasFormatProperty)) {
+                //return "Error parsing JSON file: " + file;
+                const missingScripts: string[] = [];
+
+                if (hasPrettierProperty === false) {
```

Comment:
Since `hasPrettierProperty` is a boolean, this can be cleaned up as:
```suggestion
                if (!hasPrettierProperty) {
```
Unless I'm missing something.